test/integration/local/inbound.test.js

```diff
@@ -0,0 +1,296 @@
+global.setTimeout = jest.fn()
+
+const { BlobServiceClient } = require('@azure/storage-blob')
+
+const { connect, disconnect, getClient } = require('../../../app/sftp')
+
+const { storageConfig, schemeConfig } = require('../../../app/config')
+
+const { start } = require('../../../app/polling')
+
+const { MANAGED_GATEWAY } = require('../../../app/constants/servers')
+
+const sfiDataFilename = 'SITISFI0001_AP_20231109100000.dat'
```

Comment:
Nitpick, but would it be neater to have all of these various file names in a constants folder with separate files such as sfi.js, sfiPilot.js, etc and then import the ones you want? For instance, the imps.js could contain data & control file names plus the pending equivalents, and also the return file names from outbound.test.js 

Might just make these two files less cluttered with constant setting at the top and means they are available if ever required for any other tests.

Feel free to ignore if you think this isn't adding any value